src/main/java/com/google/code/or/OpenReplicator.java

```diff
@@ -248,6 +255,21 @@ public void setBinlogEventListener(BinlogEventListener listener) {
 		this.binlogEventListener = listener;
 	}
 
+	protected void setupChecksumState() throws Exception {
+		final Query query = new Query(this.transport);
+
+		try {
+			List<String> cols = query.getFirst("SELECT @@global.binlog_checksum");

```

Comment:
Do we ever talk to the db before this in here? I mean, maybe we can make this into a simple parameter set by maxwell?